x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.test.ts

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { LogicMounter } from '../../../__mocks__';
+
+import { BoostType } from './types';
+
+import { RelevanceTuningLogic } from './relevance_tuning_logic';
+
+describe('RelevanceTuningLogic', () => {
+  const searchSettings = {
```

Comment:
Generally speaking, I renamed the overall name of this file to match its place it the web application, which is "Relevance Tuning", not "Search Settings".

The underlying model that gets updated and persisted is still named `SearchSettings` to match what we call it on the server.